crates/storage/engines/libmdbx.rs

```diff
@@ -368,6 +377,11 @@ table!(
     ( TransactionLocations ) TransactionHashRLP => (BlockNumber, Index)
 );
 
+table!(
+    /// Transaction pool trable.
+    ( TransactionPool ) TransactionHashRLP => TransactionRLP
```

Comment:
How do we query these?

Reply:
I wanted to implement the bare minimum, but yes, a fetch method should be included too!

Reply:
no worries, we can tackle in a follow up.